tensorflow/compiler/xla/service/gpu/gemm_rewriter.cc

```diff
@@ -1145,10 +1173,12 @@ class GemmRewriterVisitor : public DfsHloRewriteVisitor {
                                    HloInstruction *broadcast,
```

Comment:
As before, please comment the structure of instructions this function expects. E.g., the comment can be

```
Fuses a vector bias into a cuBLAS call. 'instr' should be an Add instruction in the following form:
    Add(OptionalBitcast(OptionalSlice(gemm)), Broadcast(OptionalConvert(...)))
'gemm' is expected to be a cuBLAS custom_call. 
```

Reply:
Resolved.